sqla_yaml_fixtures.py

```diff
@@ -1,13 +1,19 @@
 __version__ = (0, 1, 2)
 
-import functools
+try:
+    # Python 3
+    from functools import lru_cache
+except ImportError:
+    # For Python 2: 
+    # use https://pypi.python.org/pypi/backports.functools_lru_cache
+    # pip install backports.functools_lru_cache
+    from backports.functools_lru_cache import lru_cache
 
 import yaml
 import sqlalchemy
 from sqlalchemy.orm.relationships import RelationshipProperty
 
 
-
```

Comment:
Sorry, I double check and could not find anything about 3 lines after import. It seems I made that up...

While I kind of mostly follow pep8, personally I dont like this so strict blank line rules (and the checker is more strict then PEP8 itself).

First, because I use blank lines with a semantic meaning not fixed rules.
Second because other stuff I think it is not worth the trouble. 

The module level warning also seems wrong because the only thing before imports is `__version__`. 

So I would prefer not introduce this tool... I wont complain about how many blank lines you use. I just mentioned here because you made a change in area that otherwise had no changes.